templates/20-04/index.html

```diff
@@ -0,0 +1,287 @@
+      <h5>Access to Canonical partner portal</h5>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>
+        Resell the full range of Canonical's portfolio of private and public
+        cloud products: OpenStack, Kubernetes, IoT, support, security and
+        compliance for Ubuntu.
+      </p>
+      <table aria-label="Ubuntu releases and their associated architecture and kernel version">
+        <thead>
+          <tr>
+            <th>Ubuntu release</th>
+            <th>Architecture</th>
+            <th>Kernel Version</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th>
+              <strong>Ubuntu 18.04 LTS</strong>
+            </th>
+            <td>arm64, x86-64 (AMD/Intel) and s390x</td>
+            <td>5.4 (GA)</td>
+          </tr>
+          <tr>
+            <th>
+              <strong>Ubuntu 20.04 LTS</strong>
+            </th>
+            <td>arm64, x86-64  (AMD/Intel) and s390x</td>
+            <td>5.15 (HWE)</td>
+          </tr>
+        </tbody>
+      </table>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h5>Infrastructure packages</h5>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>
+        For Ubuntu 20.04 LTS, where technically feasible, Canonical provides extended security maintenance to all binary packages that reside in the Ubuntu Main Repository.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+
+      <h5>Application packages</h5>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>
+        Canonical provides application extended security maintenance to the binary packages that reside in Main and Universe. A few commonly-used packages include:
+      </p>
+      <hr class="p-rule--muted u-no-margin--bottom" />
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">Python - 3.8</li>
+        <li class="p-list__item is-ticked">PostgreSQL - 12</li>
+        <li class="p-list__item is-ticked">Ruby - 2.7</li>
+        <li class="p-list__item is-ticked">Ceph - 15.2.1</li>
+        <li class="p-list__item is-ticked">PHP - 7.4</li>
+      </ul>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {% call(slot) vf_quote_wrapper(
+    title_text="What customers say about ESM",
+    quote_size="small",
+    quote_text="ESM literally saved our lives. It's allowing us to upgrade from 14.04 LTS at our own pace. It's taken the pressure off, and it also means we can tackle the Ubuntu upgrades at the same time as we roll out the new version of our platform.",
+    citation_source_name_text="Zivago Lee",
+    citation_source_title_text="Director of DevOps Engineeringr",
+    citation_source_organisation_text="Interana",
+    is_shallow=True
+    ) -%}
+
+    {%- if slot == 'heading_link' -%}
+      <div class="p-section--shallow">
+        <a href="https://ubuntu.com/engage">All the success stories&nbsp;&rsaquo;</a>
```

Comment:
I can't find this link on the copy doc, is it missing?

Reply:
@petesfrench Took from UX wireframe, I see now it wasn't in the copydoc, so I think you're good to just delete it. I'll just double-check with Sophie